src/rest/routes/sync/index.ts

```diff
@@ -0,0 +1,74 @@
+import { Request, Response } from "express";
+import { ParamsDictionary } from "express-serve-static-core";
+import { errorWrapper } from "../../helper";
+import { Subscription } from "models/subscription";
+import { findOrStartSync } from "~/src/sync/sync-utils";
+import { determineSyncTypeAndTargetTasks } from "~/src/util/github-sync-helper";
+import { BaseLocals } from "..";
+import { RestApiError } from "~/src/config/errors";
+import { RestSyncReqBody } from "~/src/rest-interfaces";
+
+
+const restSyncPost = async (
+	req: Request<ParamsDictionary, unknown, RestSyncReqBody>,
+	res: Response<unknown, BaseLocals>
```

Comment:
Shall we type the `unknown` --> `ResponseBody` as well?

Reply:
Not sure what how it would look like

Reply:
just use void or string